Packs/CortexAttackSurfaceManagement/README.md

```diff
@@ -1,4 +1,1061 @@
-The Cortex Attack Surface Management Pack provides content for working with Attack Surface Management (ASM) data. 
+The **Cortex Attack Surface Management** Pack provides content for working with Attack Surface Management (ASM) data. 
 
 ## What does this pack do?
 - Includes playbooks for enrichment and response to alerts detected impacting your external attack surface.
+
+# Integration
+Integration to pull asset and other ASM related information
+This integration was integrated and tested with version 1.2.0 of Cortex Attack Surface Management
+
+## Configure Cortex Attack Surface Management
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Cortex Attack Surface Management.
+3. Click **Add instance** to create and configure a new integration instance.
```

Comment:
In a conversation with @altmannyarden  and @michalgold we talked about including the integration README in the pack README because integration READMEs for packs that are XSIAM only will not show up in xsoar.pan.dev.  I am willing to remove, i just wanted to follow suggestions set my Michal/Yarden.

Reply:
Spoke with @altmannyarden  and we will take the integration README out of the pack README